File: PSP/ApiRest_Request/src/main/java/dao/models/ownmodels/OwnEpisode.java
package dao.models.ownmodels;

import lombok.AllArgsConstructor;
import lombok.Getter;

import java.time.LocalDate;
import java.util.List;

@Getter
@AllArgsConstructor
public class OwnEpisode {
    private int id;
    private String name;
    private String episode;
    private LocalDate airDate;
    private List<String> characters;
}

File: Moviles/SeriesFollower/app/src/main/java/com/example/seriesfollower/domain/series/episode/OwnEpisode.kt
package com.example.seriesfollower.domain.series.episode

import java.time.LocalDate

data class OwnEpisode (
    val id:Int,
    val name:String,
    val airDate:LocalDate,
    val images: List<String>,
    val seasonNum:Int,
    val videos: List<String>,
    val castList: List<CastMember>
)